test/src/test/java/org/axonframework/test/server/AxonServerEEContainerTest.java

```diff
@@ -0,0 +1,20 @@
+package org.axonframework.test.server;
+
+import org.junit.jupiter.api.*;
+import org.testcontainers.utility.DockerImageName;
+
+/**
+ * Simple test class for AxonServerEEContainer.
+ */
+class AxonServerEEContainerTest {
```

Comment:
Would it make sense to have tests for all the public methods on the `AxonServerEEContainer` too? :thinking:

Reply:
I will try to spend some time on it and see what can be asserted properly.